roles/openshift_master/README.md

```diff
@@ -1,7 +1,7 @@
-OpenShift Master
+Cluster Master
```

Comment:
I'll update these to be in line with https://github.com/openshift/openshift-ansible/pull/443/files#diff-fd973e39d5e27e9b042867d30a3d0a99R1